Android App/app/src/main/java/org/cfp/citizenconnect/Model/User.java

```diff
@@ -23,9 +23,9 @@ public void setEmail(String _email) {
     String email;
     public  static User getUserInstance(Realm realm){
         User user = realm.where(User.class).findFirst();
-        if(user.equals(null)){
+        if(user==null){
```

Comment:
for better readability its better to have spaces with use of any operator `if(user == null)`